src/firebase/EditUserData.js

```diff
@@ -0,0 +1,38 @@
+import { db } from './config';
+
+const EditUserData = async (
+  id,
+  phoneNumber,
+  Name,
+  isAdmin,
+  Image,
+  Email,
+  coverImage,
+  canBook,
+  bookStartTime,
+  bookEndTime,
+  bookCapacity
```

Comment:
I think ( id, name, number, image, coverImage) are the only data needed here, the other ones will not be edited by the user.
and please rename the parameters with lower case for the first letter.